src/AppBundle/Resources/translations/odr-debts.en.yml

```diff
@@ -19,6 +19,7 @@ editPage:
   htmlTitle: "New deputy report - debts | GOV.UK"
   pageSectionTitle: Debts
   pageSectionDescription: Please tell us more about %client%'s debts.
+  hint: If %client% has a mortgage on a property, use the Assets section to tell us about it
```

Comment:
@lauratennant You can only add the 'hint:' below 'label:', not anywhere else. This won't get picked up by the backend.